tests/cypress/integration/notifications.cy.js

```diff
@@ -96,62 +113,143 @@ describe( 'Notifications', () => {
 			'/wp-admin/admin.php?page=' + Cypress.env( 'pluginId' ) + '#/home',
 			{ timeout: 30000 }
 		);
-
-		cy.wait( '@notifications' );
+		cy.wait( '@notifications' )
+		cy.wait( 2000 );
 	} );
 
+
+
 	it( 'Is Accessible', () => {
 		cy.injectAxe();
 		cy.wait( 1000 );
-		cy.checkA11y( '.newfold-notifications-wrapper' );
+		cy.checkA11y( '.ai-sitegen-modal__header' );
```

Comment:
It looks like this target was updated. I think it should remain as it was. It was looking at the notifications wrapper to ensure it is a11y compliant. I'm not sure why it would need to change, and I think targeting the ai sitegen modal header is not correct. I see the new notification is added, but it would load inside the existing wrapper, web plugin caught this since I don't believe it uses ai sitegen. Can we change this target  back?

Reply:
I have reverted that to '.newfold-notifications-wrapper', Also have added alt text to images which were missing it.